testing/test_lib/integration_base.py

```diff
@@ -55,6 +56,26 @@ def _ping_host(self, container, host):
         logger.debug('Return code: %s\nstderr: %s' % (return_code, str(err, 'utf-8')))
         return not return_code
 
+    def _read_yaml_from_file(self, filename):
+        with open(filename) as config_file:
+            yaml_object = yaml.load(config_file)
+        return yaml_object
+
+    def _read_faucet_config(self):
+        filename = os.path.dirname(os.path.abspath(__file__)) + \
```

Comment:
extract this to a function that just takes the argument 'forch-faucet-1/faucet/faucet.yaml' -- but everything else is in the function. It's pretty messy, and used twice, so it will help readability a fair bit.

Reply:
Done, I think.